### PR TITLE
Workbox renaming

### PR DESCRIPTION
**Pull request by gauntface**
R: @jeffposnick @addyosmani

Rename all the things!!!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 **Library Affected**:
-*sw-appcache-behavior, sw-offline-google-analytics, etc.*
+*workbox-lib, workbox-build, etc.*
```
Comment (jeffposnick) [inline .github/ISSUE_TEMPLATE.md]: `workbox-lib` -> `workbox-sw` (not that it matters)

File: packages/workbox-background-sync/demo/sw.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 /* global goog*/
 importScripts(
 	'../build/background-sync-queue.js',
-	'../../sw-routing/build/sw-routing.js',
-	'../../sw-runtime-caching/build/sw-runtime-caching.js'
+	'../../workbox-routing/build/workbox-routing.js',
+	'../../workbox-runtime-caching/build/workbox-runtime-caching.js'
```
Comment (prateekbh) [inline packages/workbox-background-sync/demo/sw.js]: shouldn't Line 17: `new goog.backgroundSync.QueuePlugin` should also be changed?

Comment (gauntface) [inline packages/workbox-background-sync/demo/sw.js]: At the moment this changes the package names and fixes build. Namespace changes I'd do in a seperate PR (just to keep this PR from ballooning further).

**Comment by addyosmani** [comment]
Going to kick travis.

**Pull request merged**